site/tests/ruleset.xml

```diff
@@ -48,7 +48,6 @@
         <exclude name="Squiz.PHP.NonExecutableCode.Unreachable" />
         <exclude name="Squiz.PHP.NonExecutableCode.ReturnNotRequired" />
         <exclude name="Squiz.PHP.NonExecutableCode.ReturnNotRequired" />
-        <exclude name="Squiz.Scope.MethodScope.Missing" />
```

Comment:
Why is this style removed from the rule set?

Reply:
It's not being removed, it's being set to not be excluded anymore as I've fixed the style errors it was reporting.

Reply:
Oh lol sorry I understood it in a weird way and thought "exclude" was like meaning "preventing `Squiz.Scope.MethodScope.Missing` from happening".

So the style checks in `ruleset.xml` are full of code that we want to get rid of in the future?

Reply:
Yeah, and that there is at least one violation of that rule in the codebase. Given the very large number of violations over a large number of rules, I decided to ignore them all, and then go one at a time through them so that:
1. the diffs are focused around a central concept making it easier to review
2. can better understand the impact of a given rule, and decide perhaps if we want to or not follow it
3. better deal with potential of fixes for a style creating a conflict with a PR

Reply:
Thanks a lot for the explanation. :)